Demos/Shopping/ShoppingUITests/TestPages/TestableUI.swift

```diff
@@ -51,7 +51,7 @@ extension TestableUI {
     ///   - line: The line of the caller
     /// - Returns: self, if successful
     @discardableResult
-    func find(_ element: XCUIElement, hittable: Bool? = nil, enabled: Bool? = nil, message: String? = nil, file: StaticString = #file, line: UInt) -> Self {
+    func find(_ element: XCUIElement, hittable: Bool? = nil, enabled: Bool? = nil, message: String? = nil, file: StaticString = #file, line: UInt = #line) -> Self {
```

Comment:
This innocuous change doesn't relate to the purpose of the PR. Nevertheless, I'm including it here because it's minor enough that it doesn't warrant its own PR.